### PR TITLE
Output formats

### PR DESCRIPTION
**Pull request by unparalleled-js**
* Fixes handling newlines
* Removed format options from `show`
* Corrects changelog entries
* Makes default send-to RAW

### CODE REVIEW

File: src/code42cli/cmds/search/extraction.py [hunk 1]
```diff
@@ -100,10 +100,7 @@ def _format_output():
         if total_events > 10 or force_pager:
             click.echo_via_pager(_format_output())
         else:
-            for page in _format_output():
-                click.echo(page, nl=False)
-            if formatter.output_format == OutputFormat.TABLE:
-                click.echo()
+            formatter.echo_formatters_output(_format_output())
```
Comment (timabrmsn) [inline src/code42cli/cmds/search/extraction.py:103]: I think you mean to use `echo_formatted_list()` here?

Comment (alanag13) [reply src/code42cli/cmds/search/extraction.py:103]: agreed, this doesn't appear to be a defined function

Comment (timabrmsn) [reply src/code42cli/cmds/search/extraction.py:103]: and we're now calling the format_output function twice on this data, which is causing it to escape the quotations and newlines

File: src/code42cli/cmds/search/extraction.py [hunk 2]
```diff
@@ -94,16 +94,10 @@ def handle_response(response):
         total_events = len(events)
         handlers.TOTAL_EVENTS += total_events
 
-        def _format_output():
-            return formatter.get_formatted_output(events)
-
         if total_events > 10 or force_pager:
-            click.echo_via_pager(_format_output())
+            click.echo_via_pager(events)
```
Comment (timabrmsn) [inline src/code42cli/cmds/search/extraction.py:98]: This should still call format on the events, otherwise it's echoing the dicts instead of json.

**Comment by alanag13** [comment]
it looks like most of our tests verify that py42 gets called with the right variables, but don't verify that anything gets output to the terminal for situations when they're supposed to. I'm not saying that we need to add those all right now because I've tested enough to know that the outputs work, but it's an observation I want to put out there because those are probably good tests to add.

**Comment by unparalleled-js** [comment]
> it looks like most of our tests verify that py42 gets called with the right variables, but don't verify that anything gets output to the terminal for situations when they're supposed to. I'm not saying that we need to add those all right now because I've tested enough to know that the outputs work, but it's an observation I want to put out there because those are probably good tests to add.

There are countless tests that check for strings inside of `results.output`.

The tests would be even better if we didn't have to support python 3.5

**Comment by alanag13** [comment]
> > it looks like most of our tests verify that py42 gets called with the right variables, but don't verify that anything gets output to the terminal for situations when they're supposed to. I'm not saying that we need to add those all right now because I've tested enough to know that the outputs work, but it's an observation I want to put out there because those are probably good tests to add.
> 
> There are countless tests that check for strings inside of `results.output`.
> 
> The tests would be even better if we didn't have to support python 3.5

You're right, thanks -- Its been a while since I spent some time in this code base so I was looking for tests that called `echo`, `echo_formatted_list`, etc and didn't see any but I see what you're talking about now.

**Comment by unparalleled-js** [comment]
I agree there are missing tests though

**Pull request merged**